src/components/InputsComponent.tsx

```diff
@@ -13,6 +14,7 @@ export default function InputsComponent({ inputState, setInputState }: InputProp
     <TruckSizeSelect onUpdate={(size) => {
       setInputState({ ...inputState, truckSize: size })
     }} truckSize={inputState.truckSize} />
+    <BehaviorSelectSliders onUpdate={(newState: Inputs) => { setInputState(newState) }} inputs={inputState} />
```

Comment:
```suggestion
    <BehaviorSelectSliders onUpdate={setInputState} inputs={inputState} />
```

This should work

Reply:
commit 52bafc73df78245010c2a9029897049786100898